storybook/stories/settings/general/General.stories.tsx

```diff
@@ -26,6 +25,61 @@ import SecuritySettings from '../../../../source/renderer/app/components/setting
 import currenciesList from '../../../../source/renderer/app/config/currenciesList.json';
 import { getLocalizedCurrenciesList } from '../../../../source/renderer/app/config/currencyConfig';
 
+interface StoryStore {
+  currentDateFormat: string;
+  currentNumberFormat: string;
+  currentTimeFormat: string;
+  currentLocale: string;
+}
+
+const mockedGeneralState = {
+  currentDateFormat: DATE_ENGLISH_OPTIONS[0].value,
+  currentNumberFormat: NUMBER_OPTIONS[0].value,
+  currentTimeFormat: TIME_OPTIONS[0].value,
+  currentLocale: LANGUAGE_OPTIONS[0].value,
+};
+
+const mockedWalletsState = {
+  currencyIsActive: true,
+  currencySelected: {
+    id: 'uniswap-state-dollar',
+    code: 'usd',
+    name: 'unified Stable Dollar',
+  },
+};
+
+const mockedSecurityStore = {
+  discreetMode: true,
+  openDiscreetMode: false,
+};
+
+const onValueChange = (
```

Comment:
It looks like the `StoryStore`, `mockedGeneralState` and the `onValueChange` are copied over from another story, right? Let's extract them to a separate file and import them in all the places ;) This file could be placed in the `stories/settings/utils` directory. I would also suggest naming them in common a way to reflect their purpose, for example: `LanguageSettingsStore`, `languageSettingsInitialState`, `onLanguageSettingsChange`.

Reply:
Right, that makes sense 🙂